### PR TITLE
Remove core dep from smoke tests

### PR DESCRIPTION
**Pull request by trask**

### CODE REVIEW

File: test/smoke/testApps/Sampling/build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apply plugin: 'war'
 
 dependencies {
-    implementation aiWebJar
+    implementation aiCoreJar
```
Comment (kryalama) [inline test/smoke/testApps/Sampling/build.gradle:4]: Is this a special case ? Still have core dependency?

Comment (trask) [reply test/smoke/testApps/Sampling/build.gradle:4]: this test uses the 2.x SDK to send telemetry, to verify that telemetry from 2.x SDK is also sampled correctly

**Review by kryalama — APPROVED** [review]

**Comment by heyams** [comment]
common schema (CS) is inside core.. now removing core dep will lead to create duplicate common schema data models in smoke/..
if we want to update CS, it needs to be updated in both places...

**Comment by heyams** [comment]
should we create a common module to avoid creating duplicate codes?

**Comment by trask** [comment]
> common schema (CS) is inside core.. now removing core dep will lead to create duplicate common schema data models in smoke/..
> if we want to update CS, it needs to be updated in both places...

sorry, I meant to explain the motivation for this change above but did not 😞

I'm removing the common schema that is inside of core as part of updating to use the Azure SDK swagger-based exporter

I want this change to go in separately, and then when we switch over to the Azure SDK swagger-based exporter, there will no need to change the smoke tests (which will be good confirmation that we didn't break anything)

**Review by heyams — APPROVED** [review]

**Pull request merged**